cmd/kubelet/app/options/options.go

```diff
@@ -438,6 +427,13 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 		mainfs.AddFlagSet(fs)
 	}()
 
+	fs.MarkDeprecated("experimental-bootstrap-kubeconfig", "Use --bootstrap-kubeconfig")
+	fs.StringVar(&c.Kubeconfig, "kubeconfig", c.Kubeconfig, "Path to a kubeconfig file, specifying how to connect to the API server. Providing --kubeconfig enables API server mode, omitting --kubeconfig enables standalone mode.")
+	fs.StringVar(&c.BootstrapKubeconfig, "experimental-bootstrap-kubeconfig", c.BootstrapKubeconfig, "deprecated: use --bootstrap-kubeconfig")
```

Comment:
How long has experimental-bootstrap-kubeconfig been deprecated? We can probably just remove the experimental version at this point.